nginx-controller/nginx/configurator.go

```diff
@@ -84,6 +84,16 @@ func (cnf *Configurator) addOrUpdateMergableIngress(mergeableIngs *MergeableIngr
 	var upstreams []Upstream
 	var keepalive string
 
+	for _, blacklistAnn := range masterBlacklist {
```

Comment:
could we have a function for filtering the master annotations?

Reply:
Done